examples/conversion_amber_gromacs/convert_to_amber_gromacs.ipynb

```diff
@@ -167,14 +194,14 @@
       "-------------------------------\n",
       "{'HarmonicAngleForce': Quantity(value=6.823125839233398, unit=kilojoule/mole),\n",
       " 'HarmonicBondForce': Quantity(value=1.4782228469848633, unit=kilojoule/mole),\n",
-      " 'NonbondedForce': Quantity(value=5127.504921744658, unit=kilojoule/mole),\n",
+      " 'NonbondedForce': Quantity(value=4.263674188305515, unit=kilojoule/mole),\n",
```

Comment:
Any thoughts as to why this changed so much?

Reply:
(It might be on my end -- Nonbonded handling wasn't implemented in a sane way until just a few days ago)

Reply:
Good catch! Let me investigate a little.

Reply:
Ah, nothing to worry about. The high energy was before the fix for #222 so probably the box vectors were too tight and the periodic distance between the two molecules was much smaller.

Reply:
Great. Thanks for checking this out!